src/screens/Notifications/NotificationScreen.tsx

```diff
@@ -0,0 +1,156 @@
+import { CompositeNavigationProp } from '@react-navigation/native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { Component } from 'react';
+import { View, Text, StyleSheet, Alert, TouchableOpacity, FlatList } from 'react-native';
+
+import { images } from 'app/assets';
+import { Header, ScreenTemplate, Button, FlatButton, ButtonType, Image } from 'app/components';
+import { Route, MainCardStackNavigatorParams, RootStackParams, Wallet } from 'app/consts';
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../loc');
+
+interface Props {
+  navigation: CompositeNavigationProp<
+    StackNavigationProp<RootStackParams, Route.MainCardStackNavigator>,
+    StackNavigationProp<MainCardStackNavigatorParams, Route.Notifications>
+  >;
+  wallets: any;
+}
+export class NotificationScreen extends Component<Props> {
+  onChangeEmailPress = () => {
+    Alert.alert('dsadasd');
```

Comment:
IMHO it's a good idea to do this, display an alert, in every place where the back-end is not ready yet or the feature will be implemented later. But I'd change the copy from "dsadasd" to something like "onChangeEmailPress" 🙃 Simply the testers feel confused when they see something like that. They almost got a heart attack when they saw Pawel's "dupa" messages 😀

Reply:
done